addons/l10n_id/data/account.account.template.csv

```diff
@@ -1,158 +1,106 @@
+l10n_id_11120001,11120001,Bank Suspense Account,account.data_account_type_current_liabilities,TRUE,l10n_id.l10n_id_chart
+l10n_id_11120002,11120002,Outstanding Receipts,account.data_account_type_current_assets,TRUE,l10n_id.l10n_id_chart
+l10n_id_11120003,11120003,Outstanding Payments,account.data_account_type_current_assets,TRUE,l10n_id.l10n_id_chart
+l10n_id_11120004,11120004,Bank,account.data_account_type_liquidity,TRUE,l10n_id.l10n_id_chart
+l10n_id_11210010,11210010,Account Receivable,account.data_account_type_receivable,TRUE,l10n_id.l10n_id_chart
+l10n_id_11210011,11210011,Account Receivable (PoS),account.data_account_type_receivable,TRUE,l10n_id.l10n_id_chart
+l10n_id_11210020,11210020,Employee Liabilities,account.data_account_type_current_assets,TRUE,l10n_id.l10n_id_chart
+l10n_id_11300180,11300180,Other Inventory,account.data_account_type_current_assets,FALSE,l10n_id.l10n_id_chart
+l10n_id_11410010,11410010,Building Rent,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11410020,11410020,Prepaid Insurance,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11410030,11410030,Prepaid Advertisement-Free,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11510010,11510010,Prepaid Tax PPh 21,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11510020,11510020,Prepaid Tax Pph 22,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11510030,11510030,Prepaid Tax Pph 23,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11510040,11510040,Prepaid Tax Pph 25,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11510050,11510050,Prepaid Tax Pph 28A,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11510060,11510060,Prepaid Tax 4 (2),account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_11800000,11800000,Down Payment,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_12210010,12210010,Office Building,account.data_account_type_fixed_assets,FALSE,l10n_id.l10n_id_chart
+l10n_id_12210020,12210020,Vehicle,account.data_account_type_fixed_assets,FALSE,l10n_id.l10n_id_chart
+l10n_id_12210030,12210030,Office Supplies,account.data_account_type_fixed_assets,FALSE,l10n_id.l10n_id_chart
+l10n_id_12281010,12281010,Accumulation Building Depreciation,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_12281020,12281020,Accumulation Vehicle Depreciation,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_12281030,12281030,Accumulation Office Supplies Depreciation,account.data_account_type_prepayments,FALSE,l10n_id.l10n_id_chart
+l10n_id_21100010,21100010,Trade Receivable,account.data_account_type_payable,TRUE,l10n_id.l10n_id_chart
+l10n_id_21100020,21100020,Shareholder Deposit,account.data_account_type_current_liabilities,FALSE,l10n_id.l10n_id_chart
```

Comment:
Missing `19999991 | Liquidity Transfer` ([Line 29 in spreadsheet](https://docs.google.com/spreadsheets/d/1nUJERIPNBTDNJEGlgjHvRWZXPq0RsncNzbfBfYtnm_M/edit#gid=109238134))

Reply:
I tried putting that before but it seemed to result to an error saying duplicate key. When I try to install the module, the 19999991 Liquidity Transfer seems to be automatically included

Reply:
Nice catch, thank you